it/FileTransmissionAcceptanceTests.scala

```diff
@@ -327,7 +327,8 @@ class FileTransmissionAcceptanceTests
       |		"checksum": "asdrfgvbhujk13579",
       |		"location": "https://localhost",
       |		"sequenceNumber": 3,
-      |		"size": 1024
+      |		"size": 1024,
+      |   "uploadTimeStamp": "2001-12-17T09:30:47Z"
```

Comment:
how is this value tested in the context of this test?

Reply:
The value itself isn't specifically tested, but since the changes made to file-transmission, it's now needed as a part of the request body to be a valid request. Otherwise the tests will fail because it's missing.